justfile

```
@@ -0,0 +1,42 @@
+# Lists all available recipes
+default:
+    just --list
+
+# Builds the project in development mode
+build:
+    cargo build $args
```

It seems that `$args` are wrong here. `just build` doesn't work on my Linux machine:
```bash
❯ just build
cargo build $args
sh: line 1: args: unbound variable
error: Recipe `build` failed on line 7 with exit code 127
```

```suggestion
    cargo build
```